Venmo/src/main/java/com/braintreepayments/api/VenmoClient.java

```diff
@@ -171,11 +254,35 @@ private Intent getLaunchIntent(Configuration configuration, String profileId) {
 
             braintreeData.put(META_KEY, meta);
             venmoIntent.putExtra(EXTRA_BRAINTREE_DATA, braintreeData.toString());
-        } catch (JSONException ignored) {}
+        } catch (JSONException ignored) {
+        }
 
         return venmoIntent;
     }
 
+    private void vaultVenmoAccountNonce(String nonce, final VenmoOnActivityResultCallback callback) {
+        VenmoAccount venmoAccount = new VenmoAccount();
+        venmoAccount.setNonce(nonce);
+
+        tokenizationClient.tokenizeREST(venmoAccount, new TokenizeCallback() {
```

Comment:
It's kind of confusing to me that we're calling `tokenizationClient.tokenizeREST` here since we're vaulting, not tokenizing. Would it make sense to call this something else?

Reply:
I guess it is still giving back a nonce, so maybe it makes sense?

Reply:
Yeah it was `TokenizationClient#tokenize()` in `v3` we kind of just went with it. We split it into two methods, `tokenizeRest` and `tokenizeGraphQL` to get rid of a dependency on `CardNonce` which is in the `Card` module now.

Reply:
`CardClient` now makes the decision on whether or not to use GraphQL.

Reply:
OK, so you don't think anyone will be confused in the future that `tokenizeREST` is actually vaulting in this case?

Reply:
I think so yeah we could re-name. I think `TokenizationClient` could be re-named here to `ApiClient` as well to mirror iOS more. Would `ApiClient#vault()` make sense?

Reply:
Sure, that would make sense to me!

Reply:
I'm OK with making that change in a separate PR if that's easier -- I know this PR needs to go out soon.

Reply:
Made a ticket!  🙌